src/platform/silabs/Logging.cpp

```diff
@@ -135,12 +138,22 @@ static void PrintLog(const char * msg)
         sz = strlen(msg);
 
 #if SILABS_LOG_OUT_UART
+#if SIWX_917
```

Comment:
Please use some more generic API and remove this `#ifdef` That's why we have a platform abstraction layer

Reply:
removed the ifdef and moved logging code to uartLogWrite API.